1223-curator_worklist/my_worklist.feature

```diff
@@ -1,146 +1,180 @@
+    When the Curator writes an answer
+    And sends it to the User
+    And the Request Type is:
+      | Approval  |
+      | DOI       |
+      | Ownership |
+    Then the User can see the answer in My Messages
+
+  @needJiraTag
+  Scenario Outline: Curator change Status on a Request
+    When they select a new status "<Status>" on a Request
+    Then the status of the Request is set to "<Status>"
+    Examples:
+      | Status   |
+      | Archived |
+      | Deleted  |
+
+  @needJiraTag
+  Scenario: Curator claims a Request
+    When the Curator do either:
+      | send an answer        |
+      | Publish               |
+      | Mint a DOI            |
+      | Decline a DOI         |
+      | Change Owner          |
+      | Decline a Ownerchange |
+    Then the Curator is stored as Owner of the Request
+
+  @needJiraTag
+  Scenario: Decide which Approval, Support or DOI Requests is in a Curators Scope
+    When the Requests' Host equals the Curators Host
+    And Curators scope on the Institution include the Requests' Users' User Profile affilliation to the Institution
+    Then the Users Request is displayed to the Curator
+
+
+  @needJiraTag
+  Scenario: Curator sees a list of Requests limited by Institution subunit scope
+    Given the Curator has a default Requests viewing scope set to a subunit of an Institution
+    When the Curator views their working list
+    Then they see only Requests of Publication owners who belong to  or subunit
+    And they can see that their viewing scope is limited to requests within the specified scope
+
+
+  @needJiraTag
+  Scenario: Curator changes their default viewing scope to another Institution subunit.
+    Given the Curator has a default Requests viewing scope set to requests for a specific Institution subunit
+    When the Curator changes their viewing scope to  another subunit of the Institution
+    Then they see only Requests of Publication owners who belong to the new subunit
+    And they can see that their viewing scope is limited to requests within the specified scope
+
+
+  @needJiraTag
+  Scenario: Curator changes their default viewing scope to include the whole Institution.
+    Given the Curator has a default Requests viewing scope set to requests for a specific Institution subunit
+    When the Curator changes their viewing scope to the whole Institution
+    Then they see all Requests of Publication owners who belong to the Institution
+    And they can see that their viewing scope is set to include all Requests in the Insitution
+
+  Scenario: Curator performs action on Request that is not in their default viewing scope
+    Given the Curator has a default Requests viewing scope set to requests for a specific Institution subunit
+    When the Curator changes their viewing scope to another Institution subunit
+    Then they see only Requests of Publication owners who belong to the new subunit
+    And they can perform all Curator related actions on these Requests
```

Comment:
I have reoriented the meaning of this scenario to remove the "not", so it is easier to understand:

Scenario: Curator performs action on Request
When the Curator opens the Work List 
And relevant Requests from Senders is listed
Then the Curator may perform actions on listed Requests